hanadb_exporter.changes

```diff
@@ -0,0 +1,8 @@
+-------------------------------------------------------------------
+Fri Jun 11 19:30:21 UTC 2021 - Bernd Schubert <bschubert@suse.com>
+
+- The HANA system replication history panel could not show data, because of 
+  the node_name variable does not match to the output string from the exporter. 
+  Changing "host=" value to "src_host=" solves the issue. Fix is tested with 
+  HANA SPS040 and HANA SPS050.
+  https://bugzilla.suse.com/show_bug.cgi?id=1187243
```

Comment:
Hi @pirat013 ,
Actually, we don't need a new changes file. We already have changes file for the dashboards at `packaging/grafana-sap-hana-dashboards`.

You would need to append there this entry. Besides this, format the bugzilla ticket as `(bgz#1187243)`, and increase the release number to `1.0.3`. Follow the same format of the current changes file.